Form/Type/FormatterType.php

```diff
@@ -154,8 +162,13 @@ public function buildView(FormView $view, FormInterface $form, array $options)
             $ckeditorConfiguration = array_merge($ckeditorConfiguration, $contextConfig);
         }
 
+        if ($this->pluginManager !== null && $this->pluginManager->hasPlugins()) {
```

Comment:
Ok, I will fix this, I followed syntax from Twig/SecurityPolicyContainerAware.php:148 being aware of Yoda syntax ;)